### PR TITLE
move static texts to locales

### PR DESCRIPTION
**Pull request by SBlazhko**

### CODE REVIEW

File: app/controllers/news_controller.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class NewsController < ApplicationController
   before_action :authenticated_only, except: :index
 
   def index
-    @all_news = News.page(params[:page]).per(5).reverse_order
+    @all_news = News.page(params[:page]).per(5).reverse_order.includes(:user)
```
Comment (IlyaUmanets) [inline app/controllers/news_controller.rb]: `includes` must be in a first order, so ` News.includes`

File: app/helpers/news_helper.rb [hunk 2]
```diff
@@ -4,6 +4,6 @@ def slice_news_text(text)
   end
 
   def news_created_label(news)
-    "Posted at #{news.created_at.strftime('%R %d-%m-%y')} by #{news.user.name}"
+    "#{t('news.index.posted')} #{news.created_at.strftime('%R %d-%m-%y')} #{t('news.index.by')} #{news.user.name}"
```
Comment (IlyaUmanets) [inline app/helpers/news_helper.rb]: It looks not clear. Can you create another helpers to connect this ?

File: app/views/news/_news_dialog.html.slim [hunk 3]
```diff
@@ -3,18 +3,18 @@
     .modal-content
       .modal-header
         button.close data-dismiss='modal' type='button' &times;
-        h4.modal-title Add news
+        h4.modal-title #{t('news.dialog.add_news')}
```
Comment (navvyua) [inline app/views/news/_news_dialog.html.slim]: h4.modal-title = t('news.dialog.add_news')

File: app/views/layouts/application.html.slim [hunk 4]
```diff
@@ -1,7 +1,7 @@
 doctype html
 html lang='en'
   head
-    title Game panel
+    title #{t('site_headers.game_panel')}
```
Comment (IlyaUmanets) [inline app/views/layouts/application.html.slim]: `title = t('site_headers.game_panel')`

File: app/views/layouts/authentication.html.slim [hunk 5]
```diff
@@ -1,7 +1,7 @@
 doctype html
 html lang='en'
   head
-    title GP - Authentication
+    title #{t('site_headers.authentication')}
```
Comment (IlyaUmanets) [inline app/views/layouts/authentication.html.slim]: Same

File: app/views/news/_news_dialog.html.slim (hunk 3 above)
Comment (IlyaUmanets) [inline app/views/news/_news_dialog.html.slim]: Same

File: app/views/news/_news_dialog.html.slim [hunk 6]
```diff
@@ -3,18 +3,18 @@
...
+              = f.text_area :text, class: 'form-control', rows: 7, required: true, placeholder: t('news.dialog.mimimun_10_symbols')
             .form-group
-              = f.label 'News photo'
+              = f.label t('news.dialog.news_photo')
               = f.file_field :news_image, required: true
         .modal-footer
-          = f.submit 'Submit', class: 'btn btn-primary'
-          button.btn.btn-default data-dismiss='modal' type='button'  Close
+          = f.submit t('news.dialog.submit'), class: 'btn btn-primary'
+          button.btn.btn-default data-dismiss='modal' type='button' #{t('news.dialog.close')}
```
Comment (navvyua) [inline app/views/news/_news_dialog.html.slim]: Same

Comment (IlyaUmanets) [inline app/views/news/_news_dialog.html.slim]: Same

File: app/views/news/index.html.slim [hunk 7]
```diff
@@ -3,10 +3,10 @@ section.content
     .col-md-12
       .box
         .box-header.with-border
-          h3.box-title News
+          h3.box-title #{t('news.index.title')}
```
Comment (IlyaUmanets) [inline app/views/news/index.html.slim]: Same

Comment (navvyua) [inline app/views/news/index.html.slim]: Same

File: app/views/news/index.html.slim [hunk 8]
```diff
@@ -3,10 +3,10 @@ section.content
...
           - if current_user_admin?
             button.btn.btn-primary.btn-sm.pull-right.btn-flat data-target='#newsModal' data-toggle='modal' type='button'
-              | Add news
+              | #{t('news.dialog.add_news')}
```
Comment (navvyua) [inline app/views/news/index.html.slim]: Same

Comment (IlyaUmanets) [inline app/views/news/index.html.slim]: same

File: spec/features/default_authentication_spec.rb [hunk 9]
```diff
@@ -13,13 +13,13 @@
 
   scenario 'user moves from sign in to sign up page' do
     visit sign_in_path
-    click_link 'Sign Up'
+    click_link 'Don\'t have an account? Sign Up here.'
```
Comment (IlyaUmanets) [inline spec/features/default_authentication_spec.rb]: `click_link t('sessions.sign_up_here')`

File: spec/features/default_authentication_spec.rb [hunk 10]
```diff
@@ -13,13 +13,13 @@
...
     expect(page).to have_button('Sign Up')
   end
 
   scenario 'user moves from sign up to sign in page' do
     visit sign_up_path
-    click_link 'Sign In'
+    click_link 'Already have an account? Sign In here.'
```
Comment (IlyaUmanets) [inline spec/features/default_authentication_spec.rb]: Same

File: app/controllers/news_controller.rb (hunk 1 above)
Comment (SBlazhko) [inline app/controllers/news_controller.rb]: fixed

File: app/helpers/news_helper.rb (hunk 2 above)
Comment (SBlazhko) [inline app/helpers/news_helper.rb]: fixed

File: app/views/news/_news_dialog.html.slim (hunk 3 above)
Comment (SBlazhko) [inline app/views/news/_news_dialog.html.slim]: fixed

File: app/views/layouts/application.html.slim (hunk 4 above)
Comment (SBlazhko) [inline app/views/layouts/application.html.slim]: fixed

File: app/views/layouts/authentication.html.slim (hunk 5 above)
Comment (SBlazhko) [inline app/views/layouts/authentication.html.slim]: fixed

File: app/views/news/_news_dialog.html.slim (hunk 3 above)
Comment (SBlazhko) [inline app/views/news/_news_dialog.html.slim]: fixed

File: app/views/news/_news_dialog.html.slim (hunk 6 above)
Comment (SBlazhko) [inline app/views/news/_news_dialog.html.slim]: fixed

Comment (SBlazhko) [inline app/views/news/_news_dialog.html.slim]: fixed

File: app/views/news/index.html.slim (hunk 7 above)
Comment (SBlazhko) [inline app/views/news/index.html.slim]: fixed

Comment (SBlazhko) [inline app/views/news/index.html.slim]: fixed

File: app/views/news/index.html.slim (hunk 8 above)
Comment (SBlazhko) [inline app/views/news/index.html.slim]: fixed

Comment (SBlazhko) [inline app/views/news/index.html.slim]: fixed

File: spec/features/default_authentication_spec.rb (hunk 9 above)
Comment (SBlazhko) [inline spec/features/default_authentication_spec.rb]: fixed

File: spec/features/default_authentication_spec.rb (hunk 10 above)
Comment (SBlazhko) [inline spec/features/default_authentication_spec.rb]: fixed

**Pull request merged**